Calibration/IsolatedParticles/interface/eHCALMatrix.h

```diff
@@ -39,6 +39,8 @@ namespace spr{
   template< typename T>
   double eHCALmatrix(const HcalTopology* topology, const DetId& det, edm::Handle<T>& hits, int ieta, int iphi, bool includeHO=false, bool algoNew=true, double hbThr=-100, double heThr=-100, double hfThr=-100, double hoThr=-100, double tMin=-500, double tMax=500, bool useRaw=false, bool debug=false);
 
+  double eHCALmatrix(const HcalTopology* topology, const DetId& det, std::vector<PCaloHit>& hits, int ieta, int iphi, bool includeHO=false, double hbThr=-100, double heThr=-100, double hfThr=-100, double hoThr=-100, double tMin=-500, double tMax=500, bool debug=false);
```

Comment:
is this specialization really needed? or should the general template be updated to take a reference to `T` instead of a handle to `T`?